examples/property_prediction/pretrain_gnns/chem/README.md

```diff
@@ -0,0 +1,21 @@
+# Strategies for Pre-training Graph Neural Networks
+
+## Intro
+This is a DGL implementation of the following paper based on PyTorch.
+
+- [Strategies for Pre-training Graph Neural Networks.](https://arxiv.org/abs/1905.12265) W. Hu*, B. Liu*, J. Gomes, M. Zitnik., P. Liang, V. Pande, J. Leskovec International Conference on Learning Representations (ICLR), 2020.
```

Comment:
I guess you need a "," or "." after "Leskovec".

Reply:
done